rasa/core/policies/ensemble.py

```diff
@@ -348,8 +348,8 @@ def probabilities_using_best_policy(
                 probabilities[
```

Comment:
@erohmensing Let's rename this probabilities for consistency as well

Reply:
okay, but how far back do we go then 😛what about L 343

Reply:
do you mean method name: `p.predict_action_probabilities`? I see your point. ok....

Reply:
I see let's leave probabilities than, but maybe add comment there that they might not be real probability distribution for now